nipype/pipeline/plugins/sgegraph.py

```diff
@@ -136,7 +136,7 @@ def make_job_name(jobnumber, nodeslist):
                     if self._qsub_args.count('-o ') == 0:
                         stdoutFile = '-o {outFile}'.format(
                             outFile=batchscriptoutfile)
-                    full_line = '{jobNm}=$(qsub {outFileOption} {errFileOption} {extraQSubArgs} {dependantIndex} -N {jobNm} {batchscript} | awk \'{{print $3}}\')\n'.format(
+                    full_line = '{jobNm}=$(qsub {outFileOption} {errFileOption} {extraQSubArgs} {dependantIndex} -N {jobNm} bash {batchscript} | awk \'{{print $3}}\')\n'.format(
```

Comment:
Forgive me if this is an obvious question but how would you do that? You can only use chmod after the file exists and the nipype creates the file just before run. Should I add an os.chmod() command to the script creation?